pkg/apis/imagepolicy/v1beta1/doc.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:deepcopy-gen=package,register
+// +k8s:conversion-gen=k8s.io/kubernetes/pkg/apis/imagepolicy
+// +groupName=imagepolicy.k8s.io
```

Comment:
yea

On Tue, Aug 9, 2016 at 4:14 PM, Q-Lee <notifications@github.com> wrote:

> In pkg/apis/imagepolicy/v1beta1/doc.go
> <https://github.com/kubernetes/kubernetes/pull/30241#discussion_r74161623>
> :
>
> > +Licensed under the Apache License, Version 2.0 (the "License");
> > +you may not use this file except in compliance with the License.
> > +You may obtain a copy of the License at
> > +
> > +    http://www.apache.org/licenses/LICENSE-2.0
> > +
> > +Unless required by applicable law or agreed to in writing, software
> > +distributed under the License is distributed on an "AS IS" BASIS,
> > +WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +See the License for the specific language governing permissions and
> > +limitations under the License.
> > +*/
> > +
> > +// +k8s:deepcopy-gen=package,register
> > +// +k8s:conversion-gen=k8s.io/kubernetes/pkg/apis/imagepolicy
> > +// +groupName=imagepolicy.k8s.io
>
> Do I need a newline?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/30241/files/a175418a03251c27316c86528b9e163c698832cf#r74161623>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAJlmzAQdrtj0ftot9w5qXP7dnz04ksJks5qeQnXgaJpZM4Jfcvq>
> .
>